doc/tooling.md

```diff
@@ -17,6 +17,10 @@ In addition to the language plugin, a linter is available to highlight errors in
 ## Gulp
 Gulp is a tool designed to help you automate and enhance your workflow. Our plugin enables you to plug the MJML translation engine into your workflow, helping you to streamline your development workflow. It is available here on [Github](https://github.com/mjmlio/gulp-mjml)
 
+## IntelliJ IDEA Plugin - MJML Support
```

Comment:
Can you put this up over gulp instead of under ?

Reply:
done :)